mbed/mbed.py

```diff
@@ -221,6 +222,13 @@ def hide_progress(max_width=80):
     if sys.stdout.isatty():
         sys.stdout.write("\r%s\r" % (' ' * max_width))
 
+def create_default_mbed_app():
+    # Default data content
+    data = {'target_overrides':{'K64F':{'platform.stdio-baud-rate': 9600}}}
```

Comment:
How would you feel about swapping the `K64F` for `*` so it would be generic for all targets?

```suggestion
    data = {'target_overrides':{'*':{'platform.stdio-baud-rate': 9600}}}
```

Reply:
@bridadan There are targets that have non-9600 default baud rates because they can't support 9600, I think.

Reply:
Ah good point (pretty sure some silicon lab targets do that).